liquidctl/driver/asetek_pro.py

```diff
@@ -0,0 +1,271 @@
+"""liquidctl drivers for sixth generation Asetek 690LC liquid coolers.
+
+Supported devices:
+
+- Corsair H100i PRO RGB, H115i PRO RGB or H150i PRO RGB (Experimental)
+
+Copyright (C) 2018–2020  Jonas Malaco and contributors
+
+Incorporates or uses as reference work by Kristóf Jakab, Sean Nelson
+and Chris Griffith.
+
+SPDX-License-Identifier: GPL-3.0-or-later
+"""
+
+import logging
+
+import usb
+
+from liquidctl.driver.asetek import Modern690Lc
+from liquidctl.driver.usb import UsbDriver
+from liquidctl.error import NotSupportedByDevice
+from liquidctl.keyval import RuntimeStorage
+from liquidctl.util import clamp
+
+
+LOGGER = logging.getLogger(__name__)
```

Comment:
Can this logger be renamed to `_LOGGER` to conform with the projects [style guide](docs/developer/style-guide.md)

Reply:
Done